### PR TITLE
[WIP] Add v3 rest client

### PR DESCRIPTION
**Pull request by justinpolygon**
Experimental change to auto generate client-php with added custom tweaks for pagination and tracing.

### CODE REVIEW

**Comment by justinpolygon** [comment]
Hey @timetraveler328, there is nothing really to review here other than the launch of the updated client. I imagine over the next few weeks we'll sand any rough edges off as people start to experiment with it.

File: LICENSE
Comment (timetraveler328) [inline LICENSE:1]: @justinpolygon we can probably add the MIT License back in so it's clear (maybe update the year too)

File: composer.json [hunk 1]
```diff
@@ -1,38 +1,41 @@
...
     "license": "MIT",
     "authors": [
         {
-            "name": "Julien Prugne",
-            "email": "julien@webeleon.dev"
-        },
-        {
-            "name": "Ersin Demirtas",
-            "email": "ed5@me.com"
+            "name": "OpenAPI",
```
Comment (timetraveler328) [inline composer.json:18]: It might make sense to have Polygon as the author so people know who to contact

**Comment by justinpolygon** [comment]
Yeah, great idea @timetraveler328! Will do.

**Review by timetraveler328 — APPROVED** [review]
I left a few comments on the license and author, but otherwise it looks okay for a beta release to me.

File: composer.json (hunk 1 above)
Comment (justinpolygon) [reply composer.json:18]: Yeah, that makes sense too. Will do. Thanks for checking this out.

File: LICENSE
Comment (justinpolygon) [reply LICENSE:1]: Fixed.

File: composer.json (hunk 1 above)
Comment (justinpolygon) [reply composer.json]: Fixed.

**Pull request merged**